test/docker/docker-compose.mpush.yml

```diff
@@ -1,5 +1,5 @@
 # This file needs to be used along with `docker-compose.mocks.yml`:
-# docker-compose -f test/docker/docker-compose.mocks.yml -f test/docker/docker-compose.mpush.yml ...
+# PRIV=priv docker-compose -f test/docker/docker-compose.mocks.yml -f test/docker/docker-compose.mpush.yml ...
```

Comment:
Why do we need this change?

Reply:
The `docker-compose-mpush.yml` requires the `PRIV` env var. It's set for us when we do `mix text.env.up` but when running from console we need to export it manually.